velox/substrait/SubstraitUtils.h

```diff
@@ -54,6 +54,9 @@ class SubstraitParser {
   /// Used to make node name in the format of n{nodeId}_{colIdx}.
   std::string makeNodeName(int nodeId, int colIdx);
 
+  /// Used to get the column index from node name.
```

Comment:
What is node name here? Does it have a special format? Would be nice to document here.

Reply:
Added the doc.